core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -368,10 +368,10 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
 
   // TODO - once workspace migration is complete and there are no more v1 workspaces or v1 billing projects, we can remove this https://broadworkbench.atlassian.net/browse/CA-1118
   private def deleteGoogleProjectIfChild(projectName: RawlsBillingProjectName, userInfoForSam: UserInfo, deleteGoogleProjectWithGoogle: Boolean = true) = {
-    def googleProjectExists(projectId: GoogleProjectId) =
+    def rawlsCreatedGoogleProjectExists(projectId: GoogleProjectId) =
```

Comment:
renamed because `googleProjectExists` is not entirely accurate with my change, which really just checks if a rawls-created google project exists